### DISCUSSION TITLE
CSV Importer: Specify default source / destination account name (instead of "(no name)")

### DISCUSSION THREAD

**Discussion by alexklibisz** [firefly-iii/firefly-iii#5353] [Firefly III] [answered]
I use the CSV importer for most of my accounts. Unfortunately none of the exported files I use include an "opposite" account column in the CSV. So when the transaction is created, it always fills in "(no name)" for the source account (on deposits) and for the destination account (on withdrawals). Then I have to update the source and destination account for each transaction. For almost all of these accounts, the opposite account is the same for all txs in a given import. For example, on credit card transactions, I use a single "Expenses" account for 99% of my expenses.

Is there any way to specify a default source and destination account for the CSV importer to use? If not, is there any chance this is on a road map for the CSV importer? I'm imagining this might work similarly to the "Default Import Account" on the `/import/configure` view.

I appreciate any input on this.

**Comment by JC5** [comment, marked as answer]
You can use rules on the Firefly III side to change these, based on the description perhaps?

**Reply by alexklibisz** [reply]
That works well enough, thanks!
